schemas/v1.0/azure.yaml.json

```diff
@@ -371,6 +370,94 @@
+                    "type": "string",
+                    "title": "The tag that will be applied to the built container image.",
+                    "description": "If omitted, a unique tag will be generated based on the format: {appName}/{serviceName}-{environmentName}:azdev-deploy-{unix time (seconds)}. Supports environment variable substitution. For example, to generate unique tags for a given release: myapp/myimage:${DOCKER_IMAGE_TAG}"
+                }
+            }
+        },
+        "aksOptions": {
+            "type": "object",
+            "title": "Optional. The Azure Kubernetes Service (AKS) configuration options",
+            "additionalProperties": false,
+            "properties": {
+                "deploymentPath": {
+                    "type": "string",
+                    "title": "Optional. The relative path from the service path to the k8s deployment manifests. (Default: manifests)",
+                    "description": "When set will override the default deployment path location for k8s deployment manifests.",
+                    "default": "manifests"
+                },
+                "namespace": {
+                    "type": "string",
+                    "title": "Optional. The k8s namespace of the deployed resources. (Default: Project name)",
+                    "description": "When specified a new k8s namespace will be created if it does not already exist"
+                },
+                "deployment": {
+                    "type": "object",
+                    "title": "Optional. The k8s deployment configuration",
+                    "additionalProperties": false,
+                    "properties": {
+                        "name": {
+                            "type": "string",
+                            "title": "Optional. The name of the k8s deployment resource to use during deployment. (Default: Service name)",
+                            "description": "Used during deployment to ensure if the k8s deployment rollout has been completed. If not set will search for a deployment resource in the same namespace that contains the service name."
+                        }
+                    }
+                },
+                "service": {
+                    "type": "object",
+                    "title": "Optional. The k8s service configuration",
+                    "additionalProperties": false,
+                    "properties": {
+                        "name": {
+                            "type": "string",
+                            "title": "Optional. The name of the k8s service resource to use as the default service endpoint. (Default: Service name)",
+                            "description": "Used when determining endpoints for the default service resource. If not set will search for a deployment resource in the same namespace that contains the service name."
+                        }
+                    }
+                },
+                "ingress": {
```

Comment:
Did we already run the `azure.yaml` configuration by PM / ux / docs? Making sure this is what we want for all developers of different audiences.

Reply:
@savannahostrowski @Austinauth - do you all have any feedback on the inline azure.yaml schema docs here?

Reply:
I think generally this looks fine to me. I left a couple of small comments.